test/models/job_test.rb

```diff
@@ -76,6 +76,11 @@
       job.command = "a\rb\r\nc\nd"
       job.commands.must_equal ["a", "b", "c", "d"]
     end
+
+    it "does not split multiline commands" do
+      job.command = "a\\\nb\nc\\\r\nd\re\\f"
```

Comment:
Would be easier to read this if you have something like:

```
job.command = <<-EO_CMD
a\
b
c\
d
e\f
EO_CMD
```

Though it does make it several lines longer.  Hmm.